src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/ReferenceAssemblies.cs

```diff
@@ -243,7 +243,11 @@ private async Task<ImmutableArray<MetadataReference>> ResolveCoreAsync(string la
                 Directory.CreateDirectory(temporaryPackagesFolder);
                 var localPathResolver = new PackagePathResolver(temporaryPackagesFolder);
 #if NET452
-                var packageExtractionContext = new PackageExtractionContext(logger);
+                var packageExtractionContext = new PackageExtractionContext(logger)
+                {
+                    PackageSaveMode = PackageSaveMode.Defaultv3,
+                    XmlDocFileSaveMode = XmlDocFileSaveMode.None,
+                };
```

Comment:
Is there a way to add a test for this case?

Reply:
It would be difficult to create a test for this case, as it would depend on the cache being empty at the start of the test, and not populated by tests running for another TFM in the same run.